scripts/build_native_lib_in_clean_repo.sh

```diff
@@ -31,7 +31,7 @@ echoerr "Moving to $PROJECT_TMP_DIR"
 cd "$PROJECT_TMP_DIR"
 
 echoerr "cloning broadinstitute/${PROJECT}"
-GIT_LFS_SKIP_SMUDGE=1 git clone git@github.com:broadinstitute/${PROJECT}.git 1>2
+GIT_LFS_SKIP_SMUDGE=1 git clone https://github.com/broadinstitute/${PROJECT}.git 1>2
```

Comment:
In my case git@ complains like that:

```
Permission denied (publickey).
fatal: The remote end hung up unexpectedly
```
 I have re-submitted fresh ssh keys to github with no luck... don't know what the problem could be. I reckon it has to do with 2FA,